operator/common/images.go

```diff
@@ -1,25 +1,29 @@
 package common
 
-type image string
+type image struct {
+	image     string
+	runAsUser int64
+}
 
-func (i image) String() string { return string(i) }
+func (i image) String() string { return i.image }
 
 type dockerhubImage image
 
 type zitadelImage image
 
-const (
-	CockroachImage       dockerhubImage = "cockroachdb/cockroach:v21.1.8"
-	PostgresImage        dockerhubImage = "postgres:9.6.17"
-	FlywayImage          dockerhubImage = "flyway/flyway:7.12.1"
-	AlpineImage          dockerhubImage = "alpine:3.11"
-	ZITADELImage         zitadelImage   = "caos/zitadel"
-	BackupImage          zitadelImage   = "caos/zitadel-crbackup"
-	ZITADELOperatorImage zitadelImage   = "caos/zitadel-operator"
+var (
+	CockroachImage        = dockerhubImage{"cockroachdb/cockroach:v21.1.8", 0}
```

Comment:
delete?
Can it be replaced by ZITADELCockroachImage in operator/database/kinds/databases/managed/adapt.go?

Reply:
theoretically yes, but I thought a differentiation between the used cockroach image and the used image to connect to cockroach should be a given

Otherwise if cockroach should run under non-root as well then their image has to be compliant as well